token/certification.go

```diff
@@ -3,37 +3,47 @@ Copyright IBM Corp. All Rights Reserved.
 
 SPDX-License-Identifier: Apache-2.0
 */
+
 package token
 
 import (
 	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
 	token2 "github.com/hyperledger-labs/fabric-token-sdk/token/token"
 )
 
+// CertificationManager is the interface for managing token certification as described by the paper
+// [`Privacy-preserving auditable token payments in a permissioned blockchain system`]('https://eprint.iacr.org/2019/1058.pdf')
 type CertificationManager struct {
```

Comment:
CertificationManager is a struct, not an interface. Does it implement some interface?

Reply:
done